### PR TITLE
change `print_escaped` to `escape_string`

### PR DESCRIPTION
**Pull request by DrKrar**
`print_escaped` is depreciated in Julia v0.5 and give error in Julia  v0.6-dev. i think this will help the pakage to run in Julia  v0.6-dev.

### CODE REVIEW

**Comment by DrKrar** [comment]
This PR is  good for windows only as the code is conditionally executed in windows.

**Comment by tkelman** [comment]
Thanks! We should probably set up appveyor for windows CI testing here

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+environment:
+  matrix:
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.4/julia-0.4-latest-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.4-latest-win64.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
+  - JULIAVERSION: "julialang/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
```
Comment (DrKrar) [inline appveyor.yml]: I think this is the correct directory as it had worked for my package [GeoEfficiency](https://github.com/DrKrar/GeoEfficiency.jl)

**Comment by bkamins** [comment]
Sorry for duplicating the above PR with #139.

**Comment by aviks** [comment]
Fixed by #147

**Pull request closed without merging**